### ISSUE TITLE
Bug: Unable to open app

### ISSUE THREAD

**Issue by blakegearin** [PradyumnaKrishna/git-icon#2]
I'm running macOS Ventura (`13.6.2`) and getting an error when trying to open the app downloaded from releases and unzipped.

<img width="284" alt="image" src="https://github.com/PradyumnaKrishna/git-icon/assets/17950836/bbccc7d1-1c0b-4d46-9908-40ca88a9b920">

Perhaps the README requirements need to be updated.

**Comment by blakegearin** [comment]
Looks like "Minimum Deployments" is set to `14.0` and if lowered I'm able to run the app.

![SCR-20231214-twuc](https://github.com/PradyumnaKrishna/git-icon/assets/17950836/8e9b98bf-cc98-4cc7-96b5-0fd57b456dcf)

It seems to correspond to `MACOSX_DEPLOYMENT_TARGET` in the `project.pbxproj` file ([source](https://github.com/PradyumnaKrishna/git-icon/blob/main/gitIcon.xcodeproj/project.pbxproj#L296)).

**Comment by PradyumnaKrishna** [comment]
> Looks like "Minimum Deployments" is set to `14.0` and if lowered I'm able to run the app.

Yeah you're right I will update the code and the release application as well. Thank you pointing out.
